Applications/VpView/vpFileDataSource.cxx

```diff
@@ -128,6 +134,57 @@ void vpFileDataSource::setMonitoringEnabled(bool enable)
   }
 }
 
+bool add_mie4nitf_subdatasets (const QString &path, QStringList &list) {
```

Comment:
Uh... your code style is rather all over the place. Please try to conform to the existing style. (n.b. `T& x`, not `T &x`, and [Allman braces](https://en.wikipedia.org/wiki/Indentation_style#Allman_style). Fortunately this is not one of the files still using historic VTK-style braces. Your indenting is also inconsistent.)